app/functions.sh

```diff
@@ -8,6 +8,25 @@
 [[ -z "${END_HEADER:-}" ]] && \
  declare -r END_HEADER='## End of configuration add by letsencrypt container'
 
+function check_nginx_proxy_container_run {
+    local _nginx_proxy_container=$(nginx_proxy_container)
+    if [[ $(docker_api "/containers/${_nginx_proxy_container}/json" | jq -r '.State.Status') = "running" ]];then
+        return 0
+    fi
+
+    echo "$(date), Error: nginx-proxy container ${_nginx_proxy_container}  isn't run" >&2
```

Comment:
`date` output should be formatted like docker-gen (`date "+%Y/%m/%d %T"`).

Minor english typo too: "nginx-proxy container `id` isn't running."

Reply:
Fixed